packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClient.kt

```diff
@@ -21,7 +22,9 @@ import org.webrtc.Logging
 
 data class Config(
```

Comment:
I'll rename it to `ConnectConfig`,just like in rn client.
WebSocketConfig wouldn't be a good name, because it's more than config for web socket (for example peer metadata)